### ISSUE TITLE
Q&V: su fragment analysis

### ISSUE THREAD

**Issue by urapadmin** [arch-kiosk/arch-kiosk-office#2382]

**Comment by urapadmin** [comment]
### kiosk 1.5.39.1
- after #2728 is solved the custom ui definition for pasu should show a proper view if you navigate to a ceramic fragment like e.g. '#038-006'. But please test a bunch of them! With images!

**Comment by urapadmin** [comment]
needs to be tested on local PASU, too!

**Comment by luizaogs** [comment]
Should the Q&V fields be Italian for this like they are in the app?

**Comment by luizaogs** [comment]
Fragment images that don't have descriptions are showing up with their uuid in Q&V. Is that purposeful? For example, for 031-003: 

<img width="417" alt="Screenshot 2024-05-27 at 3 01 56 PM" src="https://github.com/arch-kiosk/arch-kiosk-office/assets/67970402/89c0bf17-6aa3-4564-a918-c107c24e31a6">

**Comment by luizaogs** [comment]
I think it's ok otherwise. I looked at a bunch.

**Comment by luizaogs** [comment]
Oh wait. I don't know why this field is all the way over there in 035-005:

<img width="1041" alt="Screenshot 2024-05-27 at 3 07 04 PM" src="https://github.com/arch-kiosk/arch-kiosk-office/assets/67970402/1483aaec-01ec-47b9-afcd-7df1eb9b9ffb">

**Comment by luizaogs** [comment]
Ok, now I'm done.

**Comment by urapadmin** [comment]
> Oh wait. I don't know why this field is all the way over there in 035-005:
> <img alt="Screenshot 2024-05-27 at 3 07 04 PM" width="1041" src="https://private-user-images.githubusercontent.com/67970402/334218681-1483aaec-01ec-47b9-afcd-7df1eb9b9ffb.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTY4NTExNDAsIm5iZiI6MTcxNjg1MDg0MCwicGF0aCI6Ii82Nzk3MDQwMi8zMzQyMTg2ODEtMTQ4M2FhZWMtMDFlYy00N2I5LWFmY2QtN2RmMWViOWI5ZmZiLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA1MjclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNTI3VDIzMDA0MFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTVkYjZhNmRhMWNhZmRmNzc5NGMzMTk3NzkxYjg2MjE3ZmQxYjUxMWM2M2Y0MTNmZGE1ZGU2YmY3MjJlOWZmZmMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.tNKLAVwdXgMpQZVMbZgbDChyf92qzfPImZLuP69v2IU">
> 
> This also looks a bit funny, in 035-006:
> <img alt="Screenshot 2024-05-27 at 3 08 50 PM" width="1129" src="https://private-user-images.githubusercontent.com/67970402/334218898-a0788bed-3c99-4bd4-83c5-d1e9249b631b.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTY4NTExNDAsIm5iZiI6MTcxNjg1MDg0MCwicGF0aCI6Ii82Nzk3MDQwMi8zMzQyMTg4OTgtYTA3ODhiZWQtM2M5OS00YmQ0LTgzYzUtZDFlOTI0OWI2MzFiLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA1MjclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNTI3VDIzMDA0MFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTczNGU4MzA4ZTU4MGM3MGMyYTQyZjBmOWRkZGQxYWE3YmEzZTA2Y2YyZGQ2MDUyMzhjYTdiYWMzY2Q0ZTMxNWImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.KeosbqdNfFpU4g9QGkjpWAfIXWfS0cvH4F4JwfZZZ50">
> 
> Other observations is below the other field in the app, so if anything I would expect that further towards the right.
it all will get clearer with #2727: The fields manufacture /  fabric / other span two columns but don't often need them. Also if the background of the field is the same as the background the left margin makes no sense.

**Comment by urapadmin** [comment]
> Should the Q&V fields be Italian for this like they are in the app?

I thought I replied already to this? Hm. Maybe that was different ticket. No, they should not.

**Comment by urapadmin** [comment]
now that #2727 is out on Beset, the rest should look better.

**Comment by luizaogs** [comment]
> > Should the Q&V fields be Italian for this like they are in the app?
> 
> I thought I replied already to this? Hm. Maybe that was different ticket. No, they should not.

Yeah, it was a different one. Sorry.

**Comment by luizaogs** [comment]
Yes, the layout looks better.

**Comment by luizaogs** [comment]
> Fragment images that don't have descriptions are showing up with their uuid in Q&V. Is that purposeful? For example, for 031-003:
> 
> <img alt="Screenshot 2024-05-27 at 3 01 56 PM" width="417" src="https://private-user-images.githubusercontent.com/67970402/334218127-89c0bf17-6aa3-4564-a918-c107c24e31a6.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTY4NjM2MDIsIm5iZiI6MTcxNjg2MzMwMiwicGF0aCI6Ii82Nzk3MDQwMi8zMzQyMTgxMjctODljMGJmMTctNmFhMy00NTY0LWE5MTgtYzEwN2MyNGUzMWE2LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA1MjglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNTI4VDAyMjgyMlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTg4N2VhZjgwOWQxNmJmNDY2ZWI0MDZhOTM3MjdkZTMwZGFmMDc3MWFhMmY4ZTVmMWI4M2M0NDJkNzcwNjM5YjMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.oNcyLXbXtqJJcQFI93ISRVDchhjVvx1jTjKiV6FlsWQ">

The only potentially overlooked thing is this.

**Comment by urapadmin** [comment]
not so sure it it is purposeful but it might be standard view behavior. I am not sure.

**Comment by luizaogs** [comment]
Should I make that a separate ticket? I don’t think having those unintelligible strings of numbers is very helpful.

**Comment by urapadmin** [comment]
no, I'll look into it tomorrow here. Don't want the numbers to go up again :o)
